index.bs

```diff
@@ -1234,7 +1231,25 @@ intentionally mirror settable attributes and methods of the
     if a cue ID is invalid (removing an un-added ID or adding an ID twice, for example),
     the receiver may reject the text track change.
 
-Issue: Add a table for whether it's required and what the default is.
+<div class="note">
+<table>
+<tr><th>Field</th>                  <th>Default value</th>                              <th>Receiver support</th></tr>
```

Comment:
Should make it clear this is the default for the *initial* controls.  For all other controls, the default is "unchanged".

Reply:
Updated.